packages/xo-web/src/xo-app/logs/index.js

```diff
@@ -427,10 +427,7 @@ const LOG_FILTERS = {
   jobCallSkipped: '!hasErrors? callSkipped?',
 }
 
-export default [
-  propTypes({
-    jobKeys: propTypes.array,
-  }),
+const LogsCard = [
```

Comment:
Not sure about this name. @julien-f

Reply:
I think you can simply call it `Logs` because it's the default value of the `logs` module.

What do you think?

Reply:
Yes, i agree.